chain/jsonrpc-primitives/Cargo.toml

```diff
@@ -22,6 +22,7 @@ near-metrics = { path = "../../core/metrics" }
 near-primitives = { path = "../../core/primitives" }
 near-primitives-core = { path = "../../core/primitives-core" }
 near-rpc-error-macro = { path = "../../tools/rpctypegen/macro" }
+near-network = { path = "../network" }
```

Comment:
I added a bunch of rpcs that query data from PeerManager. I would prefer to keep a single copy of them, otherwise the code will get really messy if you have to have one copy for jsonrpc and one for network.

Reply:
Removed